crates/ruma-api/src/lib.rs

```diff
@@ -267,8 +267,10 @@ pub trait OutgoingRequest: Sized {
 
     /// Tries to convert this request into an `http::Request`.
     ///
-    /// This method should only fail when called on endpoints that require authentication. It may
-    /// also fail with a serialization error in case of bugs in Ruma though.
+    /// On endpoints with authentication, when adequate information isn't provided through
+    /// access_token, this could result in an error. It may also fail with a serialization error
+    /// in case of bugs in Ruma though. Finally, it may also fail if the path requested (through
+    /// `path` is not available for this endpoint.)
```

Comment:
Should update to something like "if no version in `considering_versions` supports the endpoint and it has no unstable path to fall back on".

Reply:
Oh whoops, yes, this is still reflecting the old version. :+1: 

I should give all the documentation another look, FWIW